cli.js

```diff
@@ -91,6 +98,12 @@ async function init() {
 	ora.text = 'Creating icon';
 	const composedIconPath = await composeIcon(path.join(appPath, 'Contents/Resources', `${appIconName}.icns`));
 
+	ora.text = 'Checking minimum runtime';
```

Comment:
I don't think this needs a step. We'll just show the outcome.

Reply:
Changed